src/partials/order.html

```diff
@@ -1,3 +1,42 @@
-<section class="order">
-
-</section>
\ No newline at end of file
+<section class="order-section">
```

Comment:
In section Order should be incluge **img** girl in green Vyshyvanka

Reply:
Should be add **id** in **section** `<section class="order" id="order">`

Reply:
fixed